### PR TITLE
chore(CI): Change up docker tags and notifications

### PR DESCRIPTION
**Pull request by ryanSN**
## Goal of PR
- Attempt to address some deployment items. 

@SharpNotions/ten-hour-project

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -10,22 +10,52 @@ pipeline:
...
     cleanup: true
     public: true
     pull: true
     when:
       branch: master
       event: push
+
+  build-push-prod:
+    image: plugins/docker
+    repo: sharpnotionsllc/pantry-list-frontend
```
Comment (payers1) [inline .drone.yml:37]: Isn't this the image we want to use `DRONE_BUILD_NUMBER` on? I don't think we need to build and push a new drone-now deploy image for every merge to master

Comment (ryanSN) [reply .drone.yml:37]: Thats the best practice is to deploy a new Tagged image every master merge. For a staging deploy at least.

File: .drone.yml [hunk 2]
```diff
@@ -10,22 +10,52 @@ pipeline:
...
+    when:
+      event: deployment
+      environment: [ production ]
+
+  ## TODO: Deploy to a production env.
+
+  notify:
+    image: plugins/slack
+    webhook: ${SLACK_WEBHOOK}
+    channel: zulu
```
Comment (atomdmac) [inline .drone.yml]: ☝️

Comment (ryanSN) [reply .drone.yml]: 🙏

**Review by payers1 — APPROVED** [review]

**Pull request merged**